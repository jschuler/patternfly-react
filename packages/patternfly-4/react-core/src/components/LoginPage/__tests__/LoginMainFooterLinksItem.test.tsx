// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { LoginMainFooterLinksItem } from '../LoginMainFooterLinksItem';
// any missing imports can usually be resolved by adding them here
import {  } from '..';

describe('LoginMainFooterLinksItem test', () => {
  it('LoginMainFooterLinksItem should match snapshot', () => {
    const view = shallow(
      <LoginMainFooterLinksItem
        children={<>ReactNode</>}
				href={"''"}
				target={"''"}
				className={"''"}
				linkComponent={'a'}
      />);
    expect(view).toMatchSnapshot();
  });
});
