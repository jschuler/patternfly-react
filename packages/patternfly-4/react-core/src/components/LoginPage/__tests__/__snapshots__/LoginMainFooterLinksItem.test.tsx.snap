// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginMainFooterLinksItem test LoginMainFooterLinksItem should match snapshot 1`] = `
<li
  className="pf-c-login__main-footer-links-item ''"
>
  <a
    className="pf-c-login__main-footer-links-item-link"
    href="''"
    target="''"
  >
    ReactNode
  </a>
</li>
`;
